lib/http/node.js

```diff
@@ -100,6 +100,10 @@ AWS.NodeHttpClient = AWS.util.inherit({
           total: totalBytes
         });
       });
+      if (typeof body === 'string') {
+        body = typeof Buffer.from === 'function' ?
```

Comment:
This unfortunately is broken on Node 4.4.5 :-( `Buffer.from` _is_ a function but fails (because it is actually `Uint8Array.from`)
```
root@45d8cedbf66f:/app# node --version
v4.4.5
root@45d8cedbf66f:/app# node
> Buffer.from === Uint8Array.from
true
> typeof Buffer.from
'function'
> Buffer.from("foo")
TypeError: this is not a typed array.
    at Function.from (native)
...
```